### ISSUE TITLE
Unable to load initialFile

### ISSUE THREAD

**Issue by hiaw** [pichillilorenzo/flutter_inappbrowser#56]
I was trying to load a local page. I've added the file to assets folder and specified in `pubspec.yaml`. But I was unable to load it. The following is the code I used and the error message.

        child: InAppWebView(
            initialFile: "assets/index.htm",
            initialHeaders: {},
            initialOptions: {},
            onWebViewCreated: (InAppWebViewController controller) {
              webView = controller;
            },
            onLoadError: (InAppWebViewController controller, String url,
                int code, String message) {
              print(message);
              print(code);
              print(url);
            },
          ),

```
flutter: The URL can’t be shown
flutter: 101
flutter: assets/index.html
```

**Comment by pichillilorenzo** [comment]
You wrote `initialFile: "assets/index.htm"`. Are you sure that the file extension is `htm` and not `html`?? Maybe is it a typo?

**Comment by hiaw** [comment]
I've tried both html and htm, it didn't matter.

I've create a sample app to illustrate the problem.
https://github.com/hiaw/flutter_inappbrowser_loadfile_test

**Comment by pichillilorenzo** [comment]
Thanks! I fixed this issue with new version **1.0.1**! 

However, you need to update your Android `build.gradle` file and set `minSdkVersion 17` [here](
https://github.com/hiaw/flutter_inappbrowser_loadfile_test/blob/1ab66ea72d9770fa241cfc64fa2f3a4f894b15fe/android/app/build.gradle#L37), otherwise you will have an error during Android build!
